x-pack/plugins/security_solution/public/management/cypress/cypress.d.ts

```diff
@@ -115,6 +118,12 @@ declare global {
         arg: IndexedEndpointPolicyResponse,
         options?: Partial<Loggable & Timeoutable>
       ): Chainable<null>;
+
```

Comment:
Why do we have a d.ts file? Can't we keep configuration like here in osquery? https://github.com/elastic/kibana/blob/ef375b4073983cc271a0e32ba08e719d36abd8fa/x-pack/plugins/osquery/cypress/support/e2e.ts